### PR TITLE
Release for 0.8.3

### PR DESCRIPTION
**Pull request by orta**
#trivial

### CODE REVIEW

File: danger.gemspec [hunk 1]
```diff
@@ -20,25 +20,25 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = ">= 2.0.0"
 
-  spec.add_runtime_dependency 'claide'
-  spec.add_runtime_dependency 'git'
+  spec.add_runtime_dependency 'claide', '~> 1.0'
+  spec.add_runtime_dependency 'git', '~> 0'
```
Comment (segiddins) [inline danger.gemspec]: `~> 0` does nothing

Comment (orta) [inline danger.gemspec]: that was the recommendation from `gem build` :-/ - switched them to `> 0`

Comment (orta) [inline danger.gemspec]: screen shot 2016-06-23 at 1 01 00 pm

File: danger.gemspec [hunk 2]
```diff
@@ -31,14 +31,14 @@ Gem::Specification.new do |spec|
   
   spec.add_development_dependency "bundler", "~> 1.10"
   spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec", '~> 0'
-  spec.add_development_dependency "webmock", '~> 0'
-  spec.add_development_dependency "pry", '~> 0'
+  spec.add_development_dependency "rspec", '> 0'
```
Comment (segiddins) [inline danger.gemspec]: ... this also means exactly the same thing as specifying no requirements at all

File: danger.gemspec (hunk 1 above)
Comment (segiddins) [inline danger.gemspec]: you should probably pin to a specific major version

**Pull request merged**